spec/paperclip/attachment_spec.rb

```diff
@@ -1373,14 +1373,14 @@ def call(filename)
 
       it "returns the right value when sent #avatar_fingerprint" do
         @dummy.avatar = @file
-        assert_equal 'aec488126c3b33c08a10c3fa303acf27', @dummy.avatar_fingerprint
+        assert_equal '2a979541612951c07a8a5f50228173eeb4e6ec25', @dummy.avatar_fingerprint
       end
 
       it "returns the right value when saved, reloaded, and sent #avatar_fingerprint" do
         @dummy.avatar = @file
         @dummy.save
         @dummy = Dummy.find(@dummy.id)
-        assert_equal 'aec488126c3b33c08a10c3fa303acf27', @dummy.avatar_fingerprint
+        assert_equal '2a979541612951c07a8a5f50228173eeb4e6ec25', @dummy.avatar_fingerprint
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.